test/project/validateProject.ts

```diff
@@ -89,11 +89,11 @@ export function getCSharpValidateOptions(targetFramework: string, version: FuncV
             'azureFunctions.projectLanguage': ProjectLanguage.CSharp,
             'azureFunctions.projectRuntime': version,
             'azureFunctions.preDeployTask': 'publish (functions)',
-            'azureFunctions.deploySubpath': path.join(projectSubpath ?? '', `bin/Release/${targetFramework}/publish`),
+            'azureFunctions.deploySubpath': `${projectSubpath ? `${projectSubpath}/` : ''}bin/Release/${targetFramework}/publish`,
```

Comment:
Why does `path.join` no longer work here?

Reply:
Two reasons: 

For the `deploySubpath`, we actuallly hard-code it to this directory when we initialize the project for use in VSCode. Why that's the case, I'm not sure, but @nturinski's advice was to not mess with it. Even doing something like `path.join('', 'path/to/file');` converts it to `path\\to\\file`.

For the `expectedPaths`, these are used by globby to check for matching files, and apparently globby doesn't support Windows-style path separators

Reply:
Ok. Makes sense 👍